File: fzlt245.py
array = [5, 1, 22, 25, 6, -1, 8, 10]
sequence = [1, 6, -1, -1, 10]


def isValidSubsequence(array, sequence):
    # Write your code here.
    return all(i in array for i in sequence)


print(isValidSubsequence(array, sequence))

File: fzlt245.js
const array = [5, 1, 22, 25, 6, -1, 8, 10];
const sequence = [1, 6, -1, -1, 10]

function isValidSubsequence(array, sequence) {
    // Write your code here.
    return sequence.every(r => array.includes(r))

}

console.log(isValidSubsequence(array, sequence))